src/AspNet.Security.OAuth.BungieNet/BungieNetAuthenticationConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.BungieNet;
+
+/// <summary>
+/// Contains constants specific to the <see cref="BungieNetAuthenticationHandler"/>.
+/// </summary>
+public static class BungieNetAuthenticationConstants
+{
+    public static class Claims
+    {
+        public const string ProfilePicturePath = "profilePicturePath";
+        public const string UniqueName = "uniqueName";
```

Comment:
Suggest naming these like the other custom Claim names we have ([examples](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/search?q=urn)) to start with `urn:`.